qiskit_nature/second_q/algorithms/ground_state_solvers/minimum_eigensolver_factories/vqe_uvcc_factory.py

```diff
@@ -35,8 +35,8 @@
 
 
 class VQEUVCCFactory(MinimumEigensolverFactory):
-    """Factory to construct a :class:`VQE` minimum eigensolver with :class:`UVCC` ansatz
-    wavefunction.
+    """Factory to construct a :class:`~qiskit.algorithms.VQE` minimum eigensolver with
```

Comment:
```suggestion
    """Factory to construct a :class:`~qiskit.algorithms.minimum_eigensolvers.VQE` minimum eigensolver with
```
Otherwise this points to the non-primitive version. The same needs to be changed in `VQEUCCFactory` but I cannot add a suggestion as its currently out-of-scope for the Github review

Reply:
Thanks